src/manual/using/integration/frontend.md

```diff
@@ -32,9 +32,292 @@ Taskcluster API and refresh when necessary.
 
 The
 [taskcluster-oidc-test](https://github.com/taskcluster/taskcluster-oidc-test)
-repository provides an example of a sipmle SPA following this process.  You can
+repository provides an example of a simple SPA following this process.  You can
```

Comment:
It looks like that repo is a bit behind -- not using tc-client-web, etc.  It's not really even simple.. should we still point to it?

Reply:
I'm fine with removing it.